server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -116,7 +116,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.translog.operations",
+                "es.node.stats.indices.translog.operations.count",
```

Comment:
Should be singular (operation.count)

Reply:
hmm are you sure? it actually feels ok to me

Reply:
Yes, from both our guidelines and https://opentelemetry.io/docs/specs/semconv/general/metrics/.
It could be either `operations` or `operation.count`, but since we decided for fixed suffixes, it should be the second.